aux/helpers/customer-validations/request-validation.ts

```diff
@@ -0,0 +1,59 @@
+import { Customer } from "@/domain/entities/customer-entity";
+import * as EmailValidator from 'email-validator';
+import { validateCPF } from "validations-br";
+
+export function validationRequest(body: Customer) {
+
+    if (!body.name) {
+        return `Campo nome é obrigatório! Insira um nome e tente novamente`
+    }
+
+    if (!body.cpf) {
+        return `Campo CPF é obrigatório! Insira um CPF válido e tente novamente.`
+    } else {
+        // validationCpf(body.cpf) 
+        const isValid = validateCPF(body.cpf)
+        if (isValid !== true) {
```

Comment:
To check bool values in if, you can use:

if (isValid) is equal isValid === true
if (!isValid) is equal isValid !== true